lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0-rc.2",
+  "lerna": "2.0.0-rc.5",
```

Comment:
I think you have to run npm install on the root, but lets double-check this later!